### ISSUE TITLE
grecaptcha.render is not a function

### ISSUE THREAD

**Issue by vtaits** [appleboy/react-recaptcha#233]
In formatted minified file:
![image](https://user-images.githubusercontent.com/4801414/39555110-68eda042-4e7f-11e8-90f3-ef9d12d91de4.png)

Current `grecaptcha` value:
![image](https://user-images.githubusercontent.com/4801414/39555139-859fec90-4e7f-11e8-9c06-36f1ea05440c.png)

Additional condition in `isReady` function fixes it:

```
const isReady = () => typeof window !== 'undefined' && typeof window.grecaptcha !== 'undefined' && typeof window.grecaptcha.render === 'function';
```

**Comment by cuong-nguyen** [comment]
Same here, it's just broken on our production site.

**Comment by cuong-nguyen** [comment]
@vtaits How do you make it work for you, just above condition? Where did you put it, because it's not working for me.

**Comment by angelobarot** [comment]
I thought this is only a problem in "react-google-recaptcha". Then I switched to "react-recaptcha" to try. Then viola same error. How do you fix this?

**Comment by ghengeveld** [comment]
Created a PR for this.

In our case this broke the entire app. We added an Error Boundary around it, to prevent this from happening in the future.

**Comment by appleboy** [comment]
fixed in #234
